README.md

```diff
@@ -136,6 +136,23 @@ spec:
   startup:
     # Startup has the same options as liveness
     path: ..
+  # Cascading Labels can be used if you want every resource created by your application to
+  # have the same labels, including your application. This could for example be useful for
+  # metrics, where a certain label and the corresponding resources liveliness can be combined.
+  # Any amount of labels can be added as wanted.
+  cascadingsLabels:
+    someLabel: some-label
+  # Resource Labels can be used if you want to add a label to a specific resources created by
+  # the application. One such label could for example be set on a Deployment, such that
+  # the deployment avoids certain rules from Gatekeeper, or similar. Both the Group and Kind
+  # of the resource must be specified to properly interpret which resource you want to target,
+  # group and kind are also case insensitive. Any amount of labels may be added per resourceLabel item
+  resourceLabels:
+    - resourceKind:
+        group: "apps"
+        kind: "Deployment"
+      labels:
+        someLabel: some-label
```

Comment:
Hadde det vært lettere for brukerene om vi hadde et map her?
```suggestion
  resourceLabels:
    Deployment:
      someLabel: some-label
```

Reply:
Jeg liker tanken, men er usikker på hvordan vi da kan sikre at vi får riktig group i tillegg til kind, da begge to behøves for å unikt identifisere en resource.

Reply:
Jeg tenker med intern logikk, dvs. skjule kompleksiteten fra brukerene. Vi har ingen overlapp i dag på ressurnavn av det skiperator genererer, så vi kan ha et map eller noe som inneholder ressurstypen om man skulle trenge den.

Psuedokode under
```
resources := {
  "Deployment": "apps"
}
```